projects/js-packages/ai-client/src/audio-transcription/index.ts

```diff
@@ -63,6 +60,7 @@ export default async function transcribeAudio(
 			method: 'POST',
 			body: formData,
 			headers,
+			signal: requestAbortSignal ?? undefined,
```

Comment:
I think the `?? undefined` is not necessary, as the value is undefined already it there is none.